### PR TITLE
print species dictionary with cantherm job

### PR DESCRIPTION
**Pull request by goldmanm**
prints the species dictionary for every species given with a
thermo() function in cantherm. This allows the user to 
more easily add kinetics libraries or do other manipulations
in RMG.

### CODE REVIEW

**Comment by goldmanm** [comment]
Note that this only works if the user inputs the species information directly, since the molecular structure is not input when the user uses method 1 to input species into cantherm.

**Comment by zjburas** [comment]
You can actually specify the species structure using either method 1 or 2 and it will be read in either case. I suppose this point should be clarified in the documentation.

**Comment by goldmanm** [comment]
You are completely correct, and it is clear in the documentation. I'll update the part of the documentation that I added to.

File: rmgpy/cantherm/thermo.py [hunk 1]
```diff
@@ -179,7 +179,16 @@ def save(self, outputFile):
...
         f.close()
-    
+
+        # write species dictionary
+        f = open(os.path.join(os.path.dirname(outputFile), 'species_dictionary.txt'), 'a')
+        if isinstance(species, Species):
+            if species.molecule and isinstance(species.molecule[0], Molecule):
+                f.write(species.label)
+                f.write('\n')
+                f.write(species.molecule[0].toAdjacencyList(removeH=False))
```
Comment (mliu49) [inline rmgpy/cantherm/thermo.py]: You can pass a `label` argument to `toAdjacencyList` instead of writing the label above.

**Comment by alongd** [comment]
@goldmanm, could you please rebase and review this one comment on this branch?

File: rmgpy/cantherm/thermo.py (hunk 1 above)
Comment (goldmanm) [reply rmgpy/cantherm/thermo.py]: sounds like a much better strategy. I'll fix it.

**Comment by goldmanm** [comment]
@mliu49 @alongd , Just rebased to master, and implemented the suggestion.

**Pull request merged**